F2_Cartão de Crédito/get-accounts-{creditCardAccountId}-transactions/get-credit-cards-accounts-creditCardAccountId-transactions-12.1.json

```diff
@@ -0,0 +1,357 @@
+            "identificationNumber": "0126",
+            "lineName": "SAQUE_CREDITO_BRASIL",
+            "transactionName": "SAQUE",
+            "billId": "2c30bc0e-d8e5-11eb-b8bc-0242ac130003",
+            "creditDebitType": "CREDITO",
+            "transactionType": "OUTROS",
+            "transactionalAdditionalInfo": "Saque sem tarifa",
+            "paymentType": "A_VISTA",
+            "feeType": "ANUIDADE",
+            "feeTypeAdditionalInfo": "NA",
+            "otherCreditsType": "CREDITO_ROTATIVO",
+            "otherCreditsAdditionalInfo": "NA",
+            "chargeIdentificator": "PARCELA_1",
+            "chargeNumber": 1,
+            "brazilianAmount": 50.4300,
+            "amount": 0.0000,
+            "currency": "BRL",
+            "transactionDate": "2021-07-09",
+            "billPostDate": "2021-07-09",
+            "payeeMCC": 7280
+        },
+        {
+            "transactionId": "92e219a2-d9c1-11eb-b8bc-0242ac130003",
+            "identificationNumber": "0126",
+            "lineName": "CREDITO_A_VISTA",
+            "transactionName": "TARIFAS",
+            "billId": "2c30bc0e-d8e5-11eb-b8bc-0242ac130003",
+            "creditDebitType": "DEBITO",
+            "transactionType": "TARIFA",
+            "transactionalAdditionalInfo": "NA",
+            "paymentType": "A_VISTA",
+            "feeType": "ANUIDADE",
+            "feeTypeAdditionalInfo": "NA",
+            "otherCreditsType": "CREDITO_ROTATIVO",
+            "otherCreditsAdditionalInfo": "NA",
+            "chargeIdentificator": "PARCELA_1",
+            "chargeNumber": 1,
+            "brazilianAmount": 43.0000,
+            "amount": 0.0000,
+            "currency": "BRL",
+            "transactionDate": "2021-07-08",
+            "billPostDate": "2021-07-08",
+            "payeeMCC": 4900
+        },
+        {
+            "transactionId": "6871659c-d9d5-11eb-b8bc-0242ac130003",
+            "identificationNumber": "0126",
+            "lineName": "CREDITO_A_VISTA",
+            "transactionName": "ANUIDADE",
+            "billId": "2c30bc0e-d8e5-11eb-b8bc-0242ac130003",
+            "creditDebitType": "DEBITO",
+            "transactionType": "OPERACOES_CREDITO_CONTRATADAS_CARTAO",
+            "transactionalAdditionalInfo": "NA",
+            "paymentType": "A_VISTA",
+            "feeType": "ANUIDADE",
+            "feeTypeAdditionalInfo": "NA",
+            "otherCreditsType": "CREDITO_ROTATIVO",
+            "otherCreditsAdditionalInfo": "NA",
+            "chargeIdentificator": "PARCELA_1",
+            "chargeNumber": 1,
+            "brazilianAmount": 50.0000,
+            "amount": 0.0000,
+            "currency": "BRL",
+            "transactionDate": "2021-07-08",
+            "billPostDate": "2021-07-08",
+            "payeeMCC": 4900
+        }
+    ],
+    "links": {
+        "self": "https://api.caixa.gov.br/open-banking/credit-cards-accounts/v1/accounts/b22632a4-d8e4-11eb-b8bc-0242ac130003/transactions"
```

Comment:
Pelo meu entendimento eneste endpoint é esperado todas as transações que estão vinculadas ao cartão b22632a4-d8e4-11eb-b8bc-0242ac130003 sejam apresentadas. E percebi que no arquivo `get-credit-cards-accounts-creditCardAccountId-bills-12.1.json` há outras contas com transações. Nesse caso, deveríamos refletir essas outras transações aqui também?

Reply:
Se isso for verdade, poderíamos juntar este cenário com o de paginação. Acredito que faria mais sentido.